### DISCUSSION TITLE
Inner draggable element and mobile events

### DISCUSSION THREAD

**Discussion by dellyn** [idiotWu/smooth-scrollbar#437] [Q&A] [answered]
Hello, I have a draggable element inside the scroll, and when I try to drag the element on mobile, I expect the scroll will not move, but it does. How can I stop scrolling when I'm dragging the item? 
Maybe is exist some wat to disable it or some work with events? We are using React components, last version of smooth-scrollbar
. Thanks

**Comment by dolphin-wood** [comment, marked as answer]
Try calling `preventDefault()` on touchmove/pointermove events (depending on which event you are listening to), for example:

```js
draggableEl.addEventListener('touchmove', e => {
  e.preventDefault();
});
```

**Reply by dellyn** [reply]
Yes, thank you. I fixed it with stopPropagation
